tests/hasGracefulEquipped.test.ts

```diff
@@ -32,9 +32,9 @@ describe('hasGracefulEquipped', () => {
 		expect(hasGracefulEquipped(constructGearSetup({ ...base, cape: 'Agility cape' }))).toEqual(true);
 		expect(
 			hasGracefulEquipped(constructGearSetup({ ...base, cape: 'Agility cape', head: 'Agility hood' }))
-		).toEqual(true);
+		).toEqual(false);
 		expect(hasGracefulEquipped(constructGearSetup({ ...base, cape: 'Agility cape(t)' }))).toEqual(true);
 		expect(hasGracefulEquipped(constructGearSetup({ ...base, cape: 'Max cape' }))).toEqual(true);
-		expect(hasGracefulEquipped(constructGearSetup({ ...base, cape: 'Max cape', head: 'Max hood' }))).toEqual(true);
+		expect(hasGracefulEquipped(constructGearSetup({ ...base, cape: 'Max cape', head: 'Max hood' }))).toEqual(false);
```

Comment:
these tests are fairly redundant now - instead lets add a test which ensures this particular issue doesnt happen again.

Reply:
Good call, updated.